contracts/WakuRln.sol

```diff
@@ -39,6 +60,27 @@ contract WakuRln is Ownable, RlnBase {
         }
     }
 
+    /// Register a member at specific index
+    /// @param idCommitment The idCommitment of the member
+    /// @param index The index in which to register the member
+    function registerAtIndex(uint256 idCommitment, uint256 index) external onlyOwner {
+        _validateIndexExpiration(idCommitment, index);
+        _setCommitment(idCommitment, index);
+    }
+
+    /// Renew membership credentials
+    function renew(uint256 idCommitment /*, uint256 periods*/) external onlyOwner {
+        // TODO: should this function be payable, and also, accept renewals for more than 1 period?
```

Comment:
`renew` most likely should be payable. If I remember correctly, users/accounts will have to buy themselves into the membership. So this should also apply to renewing.

However, I don't see this being done for registration. Is this on purpose?

Also, what exactly is a `period`?

Reply:
Yesterday I was talking with @rymnc about this! it seems that not having a deposit required for membership was something requested by the Waku team! 
cc: @jm-clius and @alrevuelta for their insights on this regard.

...

Regarding the `period`: I was thinking of this being a way to extend the membership for more time than the Default_TTL, i.e., if the TTL is set to 1y, then setting a `period` of 5, would mean that you would be extending your membership for 5y.